roles/openshift_prometheus/tasks/oc_apply.yaml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Checking generation of {{file_content.kind}} {{file_content.metadata.name}}
```

Comment:
@ewolinetz Isn't there a wrapper to 'oc apply' now which replaces the need for a special task?